wikidataintegrator/wdi_helpers/publication.py

```diff
@@ -281,7 +291,10 @@ def get_or_create(self, login):
         self.set_label(item)
         self.set_description(item)
 
-        success = try_write(item, self.ids['doi'], PROPS["DOI"], login)
+        if self.source == 'arxiv':
+            success = try_write(item, self.ids['arxiv'], PROPS["arxiv id"], login)
```

Comment:
I don't think arxiv entries are assigned DOIs, so this didn't make sense being hard-coded for checking the DOI